fms/models/__init__.py

```diff
@@ -285,9 +302,14 @@ def get_model(
                 devices, _guess_num_layers(lazy_sd)
             )
 
-    # Create the model
+    # TODO: detect if ckpt is unfused, customize source accordingly (?)
```

Comment:
I don't think we can create a consistent heuristic, given the names for the fused/unfused parameters are always different between each kind of model/checkpoint source/etc

Reply:
We might have to send a warning when loading a GPTQ or FP8 checkpoint that the user needs to be careful in choosing the fusing strategy